aixplain/modules/agent/__init__.py

```diff
@@ -73,6 +74,7 @@ def __init__(
         version: Optional[Text] = None,
         cost: Optional[Dict] = None,
         status: AssetStatus = AssetStatus.DRAFT,
+        tasks: Optional[List[AgentTask]] = None,
```

Comment:
Here we should initialize tasks the same way as in AgentFactory:
`tasks: List[AgentTask] = []`

Reply:
Done